README.md

```diff
@@ -234,52 +135,35 @@ Example:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/jkanyi-web/Math_Magicians/issues).
```

Comment:
Please kindly fix the broken links in your readme file. let's avoid broken links in our readme for professional repository 🛠💪